2024-03-02 07:04:57,650 INFO    MainThread:8204 [wandb_setup.py:_flush():76] Current SDK version is 0.16.3
2024-03-02 07:04:57,651 INFO    MainThread:8204 [wandb_setup.py:_flush():76] Configure stats pid to 8204
2024-03-02 07:04:57,651 INFO    MainThread:8204 [wandb_setup.py:_flush():76] Loading settings from C:\Users\16979\.config\wandb\settings
2024-03-02 07:04:57,651 INFO    MainThread:8204 [wandb_setup.py:_flush():76] Loading settings from G:\Œ“µƒ‘∆∂À”≤≈Ã\SI_630\Homeworks\Homework2\wandb\settings
2024-03-02 07:04:57,651 INFO    MainThread:8204 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-03-02 07:04:57,651 INFO    MainThread:8204 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-03-02 07:04:57,651 INFO    MainThread:8204 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-03-02 07:04:57,666 INFO    MainThread:8204 [wandb_init.py:_log_setup():526] Logging user logs to G:\Œ“µƒ‘∆∂À”≤≈Ã\SI_630\Homeworks\Homework2\wandb\run-20240302_070457-5gp6ciof\logs\debug.log
2024-03-02 07:04:57,666 INFO    MainThread:8204 [wandb_init.py:_log_setup():527] Logging internal logs to G:\Œ“µƒ‘∆∂À”≤≈Ã\SI_630\Homeworks\Homework2\wandb\run-20240302_070457-5gp6ciof\logs\debug-internal.log
2024-03-02 07:04:57,666 INFO    MainThread:8204 [wandb_init.py:init():566] calling init triggers
2024-03-02 07:04:57,666 INFO    MainThread:8204 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {}
2024-03-02 07:04:57,666 INFO    MainThread:8204 [wandb_init.py:init():591] re-initializing run, found existing run on stack: 2kz4d42m
2024-03-02 07:04:57,666 INFO    MainThread:8204 [wandb_run.py:_finish():1970] finishing run yanzhuo/attention_classifier2/2kz4d42m
2024-03-02 07:04:57,666 INFO    MainThread:8204 [jupyter.py:save_history():445] not saving jupyter history
2024-03-02 07:04:57,666 INFO    MainThread:8204 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-03-02 07:04:57,666 INFO    MainThread:8204 [wandb_init.py:_jupyter_teardown():454] cleaning up jupyter logic
2024-03-02 07:04:57,666 INFO    MainThread:8204 [wandb_run.py:_atexit_cleanup():2210] got exitcode: 0
2024-03-02 07:04:57,666 INFO    MainThread:8204 [wandb_run.py:_restore():2193] restore
2024-03-02 07:04:57,666 INFO    MainThread:8204 [wandb_run.py:_restore():2199] restore done
2024-03-02 07:05:04,602 INFO    MainThread:8204 [wandb_run.py:_footer_history_summary_info():3866] rendering history
2024-03-02 07:05:04,602 INFO    MainThread:8204 [wandb_run.py:_footer_history_summary_info():3898] rendering summary
2024-03-02 07:05:04,641 INFO    MainThread:8204 [wandb_run.py:_footer_sync_info():3825] logging synced files
2024-03-02 07:05:04,694 INFO    MainThread:8204 [wandb_init.py:init():616] starting backend
2024-03-02 07:05:04,702 INFO    MainThread:8204 [wandb_init.py:init():620] setting up manager
2024-03-02 07:05:04,752 INFO    MainThread:8204 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn, using: spawn
2024-03-02 07:05:04,752 INFO    MainThread:8204 [wandb_init.py:init():628] backend started and connected
2024-03-02 07:05:04,852 INFO    MainThread:8204 [wandb_run.py:_label_probe_notebook():1295] probe notebook
2024-03-02 07:05:04,852 INFO    MainThread:8204 [wandb_run.py:_label_probe_notebook():1305] Unable to probe notebook: 'gbk' codec can't decode byte 0xae in position 3810: illegal multibyte sequence
2024-03-02 07:05:04,852 INFO    MainThread:8204 [wandb_init.py:init():720] updated telemetry
2024-03-02 07:05:04,985 INFO    MainThread:8204 [wandb_init.py:init():753] communicating run to backend with 90.0 second timeout
2024-03-02 07:05:06,213 INFO    MainThread:8204 [wandb_run.py:_on_init():2262] communicating current version
2024-03-02 07:05:06,639 INFO    MainThread:8204 [wandb_run.py:_on_init():2271] got version response 
2024-03-02 07:05:06,639 INFO    MainThread:8204 [wandb_init.py:init():804] starting run threads in backend
2024-03-02 07:05:07,822 INFO    MainThread:8204 [wandb_run.py:_console_start():2241] atexit reg
2024-03-02 07:05:07,822 INFO    MainThread:8204 [wandb_run.py:_redirect():2096] redirect: wrap_raw
2024-03-02 07:05:07,822 INFO    MainThread:8204 [wandb_run.py:_redirect():2161] Wrapping output streams.
2024-03-02 07:05:07,822 INFO    MainThread:8204 [wandb_run.py:_redirect():2186] Redirects installed.
2024-03-02 07:05:07,839 INFO    MainThread:8204 [wandb_init.py:init():847] run started, returning control to user process
2024-03-02 07:09:44,746 WARNING MsgRouterThr:8204 [router.py:message_loop():77] message_loop has been closed
